man/systemd.exec.xml

```diff
@@ -647,11 +647,16 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
       <varlistentry>
         <term><varname>OOMScoreAdjust=</varname></term>
 
-        <listitem><para>Sets the adjustment level for the Out-Of-Memory killer for executed processes. Takes an integer
-        between -1000 (to disable OOM killing for this process) and 1000 (to make killing of this process under memory
-        pressure very likely). See <ulink
+        <listitem><para>Sets the adjustment level for the Linux kernel's Out-Of-Memory (OOM) killer for
+        executed processes. Takes an integer between -1000 (to disable OOM killing for this process) and 1000
+        (to make killing of this process under memory pressure very likely). See <ulink
         url="https://www.kernel.org/doc/Documentation/filesystems/proc.txt">proc.txt</ulink> for
-        details.</para></listitem>
+        details.</para>
+
+        <para>Use the <varname>OOMPolicy=</varname> setting of service units to configure how the service
+        manager shall react to the kernel OOM killer terminating a process of the service.  See
+        <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for details.</para></listitem>
```

Comment:
Is it normal that the default value for this setting is not documented ? man:systemd.service tells to look here for the value, but no default value is documented...

Reply:
we should probably document that, yes.